test/schema/simple_workflow.json

```
@@ -31,6 +31,7 @@
       "step2": "agent2",
       "step3": "agent1",
       "step4": "agent3"
-    }
+    },
+      "How are you?",
```

does this need to be 

```suggestion
      "prompt": "How are you?",
```

?